ground_truth/DESNAT_RPY.txt

```diff
@@ -0,0 +1,7 @@
+rpy 23 1.169040 -0.229212 0.412466 -1.603438 0.003699 1.558692
+rpy 22 -2.710969 -4.367945 0.369046 -1.576496 -0.006221 -1.588495
```

Comment:
Are these rtabmap files still relevant? The tags aren't up anymore

Reply:
nope, none of these .txts are relevant. Maybe we should delete?